install/kubernetes/helm/istio/README.md

```diff
@@ -73,33 +70,18 @@ $ helm init --service-account tiller
        app: kiali
    type: Opaque
    data:
-     username: YWRtaW4=
-     passphrase: MWYyZDFlMmU2N2Rm
+     username: `echo -n 'admin' | base64`
+     passphrase: `< /dev/urandom tr -dc 'a-zA-Z0-9' | head -c 12 | base64`
    EOF
+   echo "username is: `kubectl -n $NAMESPACE get secret/kiali -o 'go-template={{index .data "username"}}' | base64 -d` "
+   echo "passphrase is: `kubectl -n $NAMESPACE get secret/kiali -o 'go-template={{index .data "passphrase"}}' | base64 -d` "
    ```
 
 5. If you are using security mode for Grafana, create the secret first as follows:
 
 Encode username, you can change the username to the name as you want:
 ```
-$ echo -n 'admin' | base64
-YWRtaW4=
-```
-
-Encode passphrase, you can change the passphrase to the passphrase as you want:
-```
-$ echo -n '1f2d1e2e67df' | base64
-MWYyZDFlMmU2N2Rm
-```
-
-Set the namespace where Istio was installed:
-```
-$ NAMESPACE=istio-system
-```
-
-Create secret for Grafana:
-```
-$ cat <<EOF | kubectl apply -f -
```

Comment:
We still need `kubectl apply`

Reply:
sorry, missed that in 2nd secret. Fixed now.